CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ##### Enhancements
 
+* Bump Xcode version to match Xcode 13
```

Comment:
nit of nits: add 2 empty spaces at the end of this line for markdown formatting.